### ISSUE TITLE
Test git head

### ISSUE THREAD

**Issue by bastien-roucaries** [reutenauer/polyglossia#510]
Hi,

I have done some modification of polyglossia and I want more tester.

Alias could inherit now

**Comment by jspitz** [comment]
Bug: Language environments definition is broken.
```tex
% !TeX TS-program = xelatex
\documentclass{article}

\usepackage{polyglossia}
\setdefaultlanguage{english}

\begin{document}
	\begin{english}
		hello
	\end{english}
\end{document}
```

yields:
```
! LaTeX Error: Environment english undefined.
...
! LaTeX Error: \begin{document} ended by \end{english}.
```

**Comment by jspitz** [comment]
> Bug: Language environments definition is broken.

Fixed at cb468f4d94cf8ed

**Comment by jspitz** [comment]
Another bug probably introduced by this rewrite was fixed at c7489ebaf8c66.

**Comment by jspitz** [comment]
Let's close this. Polyglossia 1.54 is released and we will see what is left broken.
